walkthroughs/week-3-libraries/cloud-storage/examples/hello-world/pom.xml

```diff
@@ -12,7 +12,7 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <jetty.version>9.4.41.v20210516</jetty.version>
```

Comment:
sorry, this is an error
- [ ] Please take out the code first and there are still errors